.circleci/config.yml

```diff
@@ -98,11 +95,62 @@ jobs:
       - restore_cache:
           name: Restore hashed files if they exist
           keys:
-            - unity-editmode-2020-3-{{ checksum "../.unitysources-checksum" }}
+            - unity-build-{{ checksum "../.unitysources-checksum" }}
       - restore_cache: &RESTORE_LIBRARY_CACHE
           name: Restore library if exists
           keys:
             - library-2020-3-{{ .Branch }}
+      - run:
```

Comment:
Some of these steps were being referenced to steps in the `editmode-tests` job. Changed it so the "master" steps are in `unity-build`.